package.json

```diff
@@ -41,19 +41,20 @@
     "babel-register": "^6.26.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
-    "chai-enzyme": "^0.8.0",
+    "chai-enzyme": "^1.0.0-beta.1",
     "chalk": "^3.0.0",
     "classnames": "^2.2.6",
     "css-loader": "^3.4.2",
-    "enzyme": "^2.9.1",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.2",
     "eslint": "^6.8.0",
     "eslint-config-brainhub": "^1.12.0",
     "eslint-watch": "^6.0.1",
-    "extract-text-webpack-plugin": "^2.1.2",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
The whole PR looks good for me except using libraries in beta version what can be risky. Personally I believe we should use the latest stable release

Reply:
@RobertHebel 

the reason of using beta versions is that the latest version of one library requires a beta version of another:
- the latest `enzyme` requires beta `chai-enzyme`
- if I remember correctly, the latest `webpack` requires beta `extract-text-webpack-plugin`

Reply:
as they are dev (not prod) dependencies and beta (not alpha), IMO we can take a risk

Reply:
that it's fine by me